### PR TITLE
Adds chevron icons

### PR DESCRIPTION
**Pull request by kscherling**
Adds:

- Chevron left
- Chevron right

Adds previously existing warning icon to test array and dummy app.

### CODE REVIEW

File: app/templates/partials/-svg-icon-chevron-right.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+<path d="M5,20a0.5,0.5,0,0,1-.35-0.85l8.65-8.65L4.65,1.85a0.5,0.5,0,0,1,.71-0.71l9,9a0.5,0.5,0,0,1,0,.71l-9,9A0.5,0.5,0,0,1,5,20H5Z" transform="translate(-4.5 -1)"/>
```
Comment (vcolavin) [inline app/templates/partials/-svg-icon-chevron-right.hbs:1]: How do you write these paths? What generates them?

**Review by vcolavin — APPROVED** [review]
+1

File: app/templates/partials/-svg-icon-chevron-right.hbs (hunk 1 above)
Comment (kscherling) [reply app/templates/partials/-svg-icon-chevron-right.hbs:1]: I write them by hand ;)

Comment (kscherling) [reply app/templates/partials/-svg-icon-chevron-right.hbs:1]: Illustrator

Comment (vcolavin) [reply app/templates/partials/-svg-icon-chevron-right.hbs:1]: nice dice.

**Pull request merged**